content/panorama/layout/custom_game/common/patreon/patreon.js

```diff
@@ -141,7 +145,7 @@ function updatePaymentWindow() {
 		}
 	}
 
-	var requestData = { provider: provider, paymentKind: paymentKind };
+	var requestData = { provider: provider, paymentKind: paymentKind, paymentTargetID: paymentTargetID, paymentOriginID: Game.GetLocalPlayerID() };
```

Comment:
All events pass `PlayerID`, sending it as a data is not secure
```suggestion
	var requestData = { provider: provider, paymentKind: paymentKind, paymentTargetID: paymentTargetID };
```